src/main/java/com/mera/numbers/domain/ScheduledTask.java

```diff
@@ -0,0 +1,24 @@
+package com.mera.numbers.domain;
+
+import com.mera.numbers.service.RecordService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+@Component
+@EnableScheduling
+public class ScheduledTask {
+
+  private final RecordService recordService;
+
+  @Autowired
+  public ScheduledTask(RecordService recordService) {
+    this.recordService = recordService;
+  }
+
+  @Scheduled(fixedDelay = 5000)
+  public void printRecord() {
+    recordService.printRecord();
```

Comment:
Method should have another name: by the meaning you not only print the Record.
I think you can name it recordProcess or something like that.
Getting from API and printing will be separated after changes in RecordService.